src/api/app/views/webui2/webui/request/_show_side_links.html.haml

```diff
@@ -0,0 +1,47 @@
+%ul.list-unstyled
+  %li
+    %i.fas.fa-user.text-success
+    Created by
+    = user_with_realname_and_icon(request_creator, no_icon: true, short: true)
+    = fuzzy_time(request_created_at)
+  %li
+    %i.fa.fa-info-circle.text-info
+    In state
+    = link_to(request_state, { anchor: 'request_history' }, style: "color: #{request_state_color(request_state)};")
+  - unless package_maintainers.empty?
+    %li
+      %i.fas.fa-eye.text-info{ title: 'Look into' }
+      = pluralize(package_maintainers.size, 'package maintainer')
+      (#{link_to('show who', '#', class: 'show_dialog', data: { toggle: 'modal', target: '#show-maintainers-modal' })})
```

Comment:
The link `(show who)` is a bit weird. I would do this instead:

 ```
= link_to('#', class: 'show_dialog', data: { toggle: 'modal', target: '#show-maintainers-modal' }) do
  = pluralize(package_maintainers.size, 'package maintainer')
```

It would look like this
![suggestion](https://user-images.githubusercontent.com/1102934/52068447-fa7a4400-257c-11e9-9eaf-6e6a8d1200e1.png)

Reply:
done